programs/cleos/main.cpp

```diff
@@ -2439,29 +2439,21 @@ int main( int argc, char** argv ) {
    string table;
    string lower;
    string upper;
-   string table_key;
-   string key_type;
    string encode_type{"dec"};
    bool binary = false;
    uint32_t limit = 10;
-   string index_position;
+   string index;
    bool reverse = false;
    bool show_payer = false;
    auto getTable = get->add_subcommand( "table", localized("Retrieve the contents of a database table"), false);
    getTable->add_option( "account", code, localized("The account who owns the table") )->required();
    getTable->add_option( "scope", scope, localized("The scope within the contract in which the table is found") )->required();
    getTable->add_option( "table", table, localized("The name of the table as specified by the contract abi") )->required();
+   getTable->add_option( "index", index, localized("Index name. The same as in abi decription\n"))->required();
    getTable->add_option( "-b,--binary", binary, localized("Return the value as BINARY rather than using abi to interpret as JSON") );
    getTable->add_option( "-l,--limit", limit, localized("The maximum number of rows to return") );
-   getTable->add_option( "-k,--key", table_key, localized("Deprecated") );
    getTable->add_option( "-L,--lower", lower, localized("JSON representation of lower bound value of key, defaults to first") );
    getTable->add_option( "-U,--upper", upper, localized("JSON representation of upper bound value of key, defaults to last") );
-   getTable->add_option( "--index", index_position,
-                         localized("Index number, 1 - primary (first), 2 - secondary index (in order defined by multi_index), 3 - third index, etc.\n"
-                                   "\t\t\t\tNumber or name of index can be specified, e.g. 'secondary' or '2'."));
-   getTable->add_option( "--key-type", key_type,
```

Comment:
It is better to keep not-used options and show in description that they are `deprecated`